pterodactyl.php

```diff
@@ -176,9 +178,9 @@ public function addService(
         // Get the service helper
         $this->loadLib('pterodactyl_service');
         $service_helper = new PterodactylService();
-        $pterodactyl_user = $this->apiRequest('Users', 'getByExternalId', [$vars['client_id']]);
         if ($vars['use_module'] == 'true') {
             // Load/create user account
+            $pterodactyl_user = $this->apiRequest('Users', 'getByExternalId', [$vars['client_id']]);
```

Comment:
The external ID probably should also be something more descriptive/unique than the client's ID, e.g. `"blesta-" . $vars['client_id']`